.editorconfig

```diff
@@ -0,0 +1,21 @@
+# http://editorconfig.org/
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+
+[*.go]
+indent_style = tab
+
+[*.{js,jsx,json,html}]
```

Comment:
Mind adding `md` to this?

Reply:
Will do, and I'll also add that in the other repositories as well

Reply:
Added it on it's one line since I needed to disable `trim_trailing_whitespace` for Markdown files since they need it sometimes